app/services/idv/actions/verify_document_action.rb

```diff
@@ -0,0 +1,41 @@
+module Idv
+  module Actions
+    class VerifyDocumentAction < Idv::Steps::DocAuthBaseStep
+      def call
+        enqueue_job
+      end
+
+      private
+
+      def form_submit
+        json = form.submit
+        render_json(json, status: form.status)
+        json
+      end
+
+      def form
+        @form ||= Idv::ApiDocumentVerificationForm.new(
+          params,
+          liveness_checking_enabled: liveness_checking_enabled?,
+        )
+      end
+
+      def enqueue_job
+        document_capture_session = create_document_capture_session(
+          verify_document_capture_session_uuid_key,
+        )
+        document_capture_session.requested_at = Time.zone.now
+        document_capture_session.store_proofing_pii_from_doc({})
+
+        VendorDocumentVerificationJob.perform(
+          _document_capture_session_uuid: params[:document_capture_session_uuid],
```

Comment:
I think we are preferring to send the `document_capture_session.result_id` to the lambda for use in the callback since it never gets shared client-side like the uuid